examples/blank-typescript/package.json

```diff
@@ -12,6 +12,7 @@
     "@botonic/react": "^0.30.0"
   },
   "devDependencies": {
-    "@botonic/dx": "^0.30.0"
+    "@botonic/dx": "^0.30.0",
+    "@botonic/dx-bundler-rspack": "^0.30.0-alpha.0"
```

Comment:
we have released version 0.30.0-alpha.4 but by default it installs alpha.0 because it is marked as latest